src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use './variables' as *;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: $star-size;
+    width: $star-size;
+    margin-right: $star-margin;
```

Comment:
It's good to see that you are using variables for the star size and margin. This promotes consistency and makes it easier to maintain. Ensure that these variables are defined with appropriate values in the 'variables.scss' file.